### PR TITLE
Hide RSVP tags for events with zero attendees (#190)

### PR DESCRIPTION
**Pull request by dorono**
Looks like the RSVP badges are not in prod yet, but in any event, locally, here is what I have.

**NOTE** - Tried adding a test for RSVP badge for event details using same approach as event card, but didn't work, and didn't have time to debug, so would love to get any feedback on that (still sort of a Jest n00b). 
eventlist
eventdetail

### CODE REVIEW

File: src/components/EventCard/EventCard.js [hunk 1]
```diff
@@ -55,7 +55,9 @@ const EventCard = ({ event, className }) => {
           </Link>
         </div>
 
-        <RsvpBadge totalAccepted={totalAccepted} center />
+        { totalAccepted > 0 &&
```
Comment (pdw207) [inline src/components/EventCard/EventCard.js]: Can you put totalAccepted on the next line?

File: src/components/EventDetails/EventDetails.js [hunk 2]
```diff
@@ -110,9 +110,11 @@ class EventDetails extends Component {
               endDate={endDate}
             />
 
+            { totalAccepted > 0 &&
```
Comment (pdw207) [inline src/components/EventDetails/EventDetails.js]: next line

Comment (dorono) [reply src/components/EventDetails/EventDetails.js]: Done!

File: src/components/EventCard/EventCard.js (hunk 1 above)
Comment (dorono) [reply src/components/EventCard/EventCard.js]: Done!

**Pull request merged**